### ISSUE TITLE
Clock_ext on v4sa

### ISSUE THREAD

**Issue by MedourMehdi** [mikrosk/atari_sound_setup#2]
Hi,

Just for information, this is [how clock_ext](https://github.com/MedourMehdi/mm_mint_pic/blob/main/utils_snd/utils_snd.cpp#L60) is handled on v4sa in case you'll want to support it.

Regards,

**Comment by mikrosk** [comment]
Thanks. As far as I can see, it's basically Falcon compatible. I guess I'll have to either think of a (precise) way how to measure the clocks on other than Falcon (so I don't access hardware directly) or introduce hardcoded external clocks in case a specific machine (like V4SA) is detected.

**Comment by MedourMehdi** [comment]
Yes OL tried to make it that way, but there's a lot of emulation done via [v4sa emutos](https://sourceforge.net/p/v4sa-emutos/code/HEAD/tree/trunk/) - I used an arbitrary value for clock_value in my code, just in order to compute the prescale value. For example the 44.1khz frequency play in real (on v4sa) at 44.312khz and for 48Khz we have 48,561khz.
There're also some asm example [here](http://www.apollo-core.com/index.htm?page=coding&tl=3) regarding how to play sound on v4sa.
